engineering/learning/os/command-line.md

```diff
@@ -0,0 +1,7 @@
+For Paid Courses please speak with Mentors / Team leads as we may have copies available.
+
+## Command line:
+
+- [https://commandlinepoweruser.com/](https://commandlinepoweruser.com/) - Every developer should go through this video series to help them master their command line and improve their workflow.
+
+- [The Art of Command Line](https://github.com/jlevy/the-art-of-command-line) - A really great resource for Bash and command line in general. Fluency in the command line is a necessity for any engineer. An engineer will be severely limited and handicapped without it. This article is not just for beginners. It covers some of the more obscure features in bash and command line.
```

Comment:
@kaiomagalhaes can we replace `handicapped` with a more appropriate word?

Reply:
@alesmit I went ahead and removed the whole sentence since it doesn't add to the message